app/cron.yaml

```diff
@@ -30,3 +30,7 @@ cron:
 - description: clean up in test mode
   url: /global/tasks/clean_up_in_test_mode
   schedule: every 60 minutes
+
+# Send email notification when there are many unreviewed records.
+- description: send email notification for review
+  url: /global/tasks/notify_bad_review_status
```

Comment:
Optional: The task name looks a bit vague. Something like "notify_many_unreviewed_notes" may be better.

Also, add a line break in the end of the line.